css/style.css

```
@@ -16,4 +15,98 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+nav {
+  padding: 20px 40px 3px 60px;
+  background-color: rgb(247, 244, 241);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+}
+.nav_link > img {
+  width: 50px;
+  height: 40px;
+}
+
+.nav_link ul,
+li {
+  display: flex;
+  font-size: 0.8rem;
+  padding-left: 15px;
+}
+li:active {
+  color: red;
+}
+.main-container {
+  background-image: url(/img/first-background.jpg);
+  height: 350px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 0.8rem;
+  line-height: 15px;
+}
+.main-container>button{
+
+    background-color: red;
+    border: none;
+  padding:5px 10px;
```

```suggestion
  padding: 0.3rem 0.6rem;
```
We should use relative units like rem instead of absolute ones like px